src/main/java/liuni/TwitterStatus.java

```diff
@@ -11,19 +10,12 @@
 
     private static final int TWITTER_CHAR_MAX = 280;
 
-    public void postStatus(String text) {
+    public void postStatus(String text) throws Exception {
```

Comment:
this does work.  Can you make the endpoint scale for multiple input fields?

Reply:
Multiple input fields such as? I am not sure I fully understand. Like number only type of input?